static/src/js/routes/Data/Data.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import {
+  Link,
+  Route,
+  Switch,
+  withRouter
+} from 'react-router-dom'
+
+import actions from '../../actions/index'
+
+import OrderStatusContainer from '../../containers/OrderStatusContainer/OrderStatusContainer'
+import SecondaryToolbarContainer
+  from '../../containers/SecondaryToolbarContainer/SecondaryToolbarContainer'
+
+const mapStateToProps = state => ({
+  order: state.order
+})
+
+const mapDispatchToProps = dispatch => ({
+  onChangePath:
+    newHeight => dispatch(actions.masterOverlayPanelResize(newHeight))
+})
+
+export const Data = ({
+  order = {},
+  onChangePath
+}) => {
+  const { jsondata = {} } = order
+  const { source } = jsondata
+  return (
+    <div className="route-wrapper route-wrapper--dark route-wrapper--content-page">
+      <div className="route-wrapper__content">
+        <header className="route-wrapper__header">
+          <SecondaryToolbarContainer />
+        </header>
+        <nav className="route-wrapper__content-nav">
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
Is it better to do this or to provide the href with `#` or something?

Reply:
Good catch. This is a leftover from when I was doing that. Its no longer needed when using the `Link` component